config/crd/bases/koku-metrics.openshift.io_kokumetrics.yaml

```diff
@@ -6,20 +6,20 @@ metadata:
   annotations:
     controller-gen.kubebuilder.io/version: v0.3.0
   creationTimestamp: null
-  name: costmanagements.cost-mgmt.openshift.io
+  name: kokumetrics.koku-metrics.openshift.io
 spec:
-  group: cost-mgmt.openshift.io
+  group: koku-metrics.openshift.io
   names:
-    kind: CostManagement
-    listKind: CostManagementList
-    plural: costmanagements
-    singular: costmanagement
+    kind: KokuMetrics
```

Comment:
I don't think the kind should be plural because then the CRD shows up as a `KokuMetrics` I think we should name it `KokuMetricConfig` or something similar.

Reply:
Changed it to `KokuMetricsConfig`